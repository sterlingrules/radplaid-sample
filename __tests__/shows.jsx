import React from 'react'
import { configure, shallow } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'
import Shows from './../src/app/shows/index.jsx'

configure({ adapter: new Adapter() })

describe('Shows', () => {
	it('renders correctly', () => {
		expect(shallow(<Shows />)).toMatchSnapshot()
	})
})
