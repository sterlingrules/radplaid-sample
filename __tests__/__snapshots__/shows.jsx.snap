// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Shows renders correctly 1`] = `
<Route>
  <Component />
</Route>
`;
